vtex/billing-app
# Update/contract

#### What is the purpose of this pull request?


#### What problem is this solving?
Billing system is using the field AppId in its contracts to make queries and other operations, while the Name field is used for display in the UI. This update populates the AppId field without changing the Name field, so no breaking changes occurr for now

#### How should this be manually tested?
1. Clone/Pull this repo/branch and link in your workspace
2. Try to install a paid app, e.g. `vtex install vtex.sms-provider -r vtex --verbose`, or purchase the SMS Provider in the Extension Store
3. This app is already bought for VTEX accounts, so it should display the `Installed from previous purchase` message, while installing successfully in the CLI.


#### Screenshots or example usage

#### Types of changes
